### ISSUE TITLE
Certain resources do not update when enabling/disabling mods due to being cached on startup.

### ISSUE THREAD

**Issue by TaigaTreant** [Meowlala/RepentanceAPIIssueTracker#35]
Certain resources (such as UI graphics) are cached on startup, and can only be affected when initially loading the game. As such, enabling/disabling mods that affect these resources cause no effect until restarting the game, which can be confusing to some users.

**Comment by Meowlala** [comment]
Out of curiosity, are these the same resources marked in `preload.xml`?

**Comment by TaigaTreant** [comment]
I don't believe so. At the least, if those are affected, then it's not a complete list (the game over screen, for example, is affected but none of it's resources are listed there).

**Comment by UnderscoreKilburn** [comment]
This should have been at least partially fixed by the last patch, any resources still affected by this?
